File: Python/For e While/6.py
#Prova 2 do JUDE
#Questão 3

cor_anterior = ""
cor_atual = input()

while True:
    if cor_anterior != "":
        if cor_anterior == cor_atual:
            print (cor_anterior)
        elif (cor_anterior == "azul" and cor_atual == "verde") or (cor_atual == "azul" and cor_anterior == "verde"):
            print ("ciano")
        elif (cor_anterior == "amarelo" and cor_atual == "verde") or (cor_atual == "amarelo" and cor_anterior == "verde"):
            print ("lima")
        elif (cor_anterior == "vermelho" and cor_atual == "verde") or (cor_atual == "vermelho" and cor_anterior == "verde"):
            print ("marrom")
        elif (cor_anterior == "vermelho" and cor_atual == "azul") or (cor_atual == "vermelho" and cor_anterior == "azul"):
            print ("roxo")
        elif (cor_anterior == "azul" and cor_atual == "amarelo") or (cor_atual == "azul" and cor_anterior == "amarelo"):
            print ("verde")
        elif (cor_anterior == "vermelho" and cor_atual == "amarelo") or (cor_atual == "vermelho" and cor_anterior == "amarelo"):
            print ("laranja")
    if cor_atual == 'preto':
        print('Game Over')
        break
    elif cor_atual == 'branco':
        break
    cor_anterior = cor_atual
    cor_atual = input()

File: Cpp/Vetores/6.cpp
//Lista 3 do JUDE
//Questão 6

#include <iostream>
using namespace std;

int main() 
{
  int N, C, X;
  cin >> N;
  int V[N];
  
  for (int i = 0; i < N; i++)
  {
  	cin >> X;
  	V[i] = X;
  }
  
  cin >> C;
  
  for (int i = 0; i < N; i++)
  {
  	if (V[i] == C)
  		{
  		cout << C << endl;
  		break;
  		}
  	if (i == (N-1))
  		{
  		cout << "-1" << endl;
  		}
  }
  return 0;
}
